contents.json

```diff
@@ -1598,109 +1598,129 @@
   "widgets": {
     "2x2": [
       {
+        "name": "Image widget with bottom titles",
```

Comment:
I think it's not necessary to have the word "widget" in every name. Can we remove it? I mean in general, names should be descriptive and searchable. The user knows that he is inserting a widget.